www/data_service/src/md_demo/demo.controller.coffee

```diff
@@ -0,0 +1,6 @@
+class Demo extends Controller('app')
+    constructor: ($scope, $timeout, $log, dataService) ->
+        opened = dataService.open()
+        opened.closeOnDestroy($scope)
+        opened.getBuilders().then (@builders) =>
+            @builders[0].loadBuilds(limit: 10)
```

Comment:
Will `loadBuilds` return a promise? Can we use `$scope.builds = @builders[0].loadBuilds()` to bind the builds to a variable?